docker-compose/docker-compose.yml

```
@@ -0,0 +1,99 @@
+    ports:
+      - "8060:8060"
+    healthcheck:
+      test: "curl --fail --silent localhost:8060/actuator/health/readiness | grep UP || exit 1"
+      interval: 10s
+      timeout: 5s
+      retries: 10
+      start_period: 5s
+    extends:
+      file: common-config.yml
+      service: microservice-base-config
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)